Formula/fuse-ext2.rb

```diff
@@ -0,0 +1,67 @@
+class FuseExt2 < Formula
+  desc "FUSE module to mount ext2/3/4 file systems with read write support"
+  homepage "https://github.com/alperakcan/fuse-ext2"
+  url "https://github.com/alperakcan/fuse-ext2/archive/v0.0.10.tar.gz"
+  sha256 "ad2260df4ccfb8ba9f761c66ea7c3b24bf690eab46e6d562d2d2e5a5f2f76dff"
+  head "https://github.com/alperakcan/fuse-ext2.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on :osxfuse
+  depends_on "e2fsprogs"
+
+  conflicts_with "ext2fuse"
+
+  def install
+    ENV.append "LIBS", "-losxfuse"
+    ENV.append "CFLAGS", "-idirafter/usr/local/include/osxfuse/fuse"
+    ENV.append "CFLAGS", "--std=gnu89" if ENV.compiler == :clang
```

Comment:
I don't see no reason why this would be necessary (and it's not indicated by upstream)